src/data/projects.ts

```diff
@@ -170,9 +170,41 @@ const PROJECT_DESCRIPTORS: ProjectDescriptor[] = [
                     orientation: "landscape"
                 },
                 description: "Nearly finished at 23/25 repeats. Here I folded the scarf in half and compared with another scarf I had made. The blue velvet scarf is a little shorter",
-            }
+            },
+            
         ],
-    }
+    },
+    {
+        id: "stretchy-blocks",
+        title: "Stretchy Blocks",
+        years: "2024",
+        sort_key: "2024-02-00:03",
+        alt_text: "Screenshot of the Stretchy Blocks animation",
+        img_format: "png",
+        description: `
+        <div class="warning">
+            ⚠ Compatibility Note: The live demo linked above requires WebGPU. This is a recent
+            browser feature, and not all browsers support it. As of 2024-02-15,
+            this is known to work in Chrome and Edge for Windows, but not Firefox or
+            on mobile devices. Check <a href="https://caniuse.com/webgpu">Can I use...</a>
+            for up-to-date compatibility information.
```

Comment:
Whenever I next use WebGPU, I should move this message elsewhere and add a project field. Perhaps call it `accessibility_notes: ['requires_webgpu']`. But until that happens, it's overkill.